crates/zoon/src/style/transitions.rs

```diff
@@ -3,13 +3,31 @@ use std::borrow::Cow;
 
 // ------ Transitions ------
 
+/// Define transitions for an element between two states.
+/// More information at <https://developer.mozilla.org/en-US/docs/Web/CSS/transition>
 #[derive(Default)]
 pub struct Transitions<'a> {
+    /// Static css properties used by zoon.
     static_css_props: StaticCSSProps<'a>,
+    /// Customizable css properties which can be added.
     dynamic_css_props: DynamicCSSProps,
 }
 
 impl<'a> Transitions<'a> {
+    /// Apply a transformation to an element. For now we need to use raw css to
+    /// update the element with the use of `pseudo class`
```

Comment:
I used the examples from https://developer.mozilla.org/en-US/docs/Web/CSS/transition and I did not find out other example in mind I could relate too. What kind of additional example could I write ?

Reply:
Perhaps we can just remove the sentence `For now we need to use raw css to
    /// update the element with the use of pseudo class` altogether. I'm not sure we need more examples.
Also don't kill too much time with `Transitions` because there is big chance I'll deprecate them in the near future.

Reply:
Okay :wink: